regtest/funnel/rt-fps/test.dat

```diff
@@ -0,0 +1,8 @@
+WHOLEMOLECULES ENTITY0=54,75,212,228,239,258,311,328,348,372,383,402,421,463,487,503,519,657,674,690,714,897,914,934,953,964,974,985,1007,1018,1037,1247,1264,1283,1302,1324,1689,1708,1727,1738,1962,1984,1994,2312,2329,2349,2467,2490,2500,2517,2524,2536,2547,2554,2569,2575,2591,2607,2635,2657,2676,2693,2700,2719,2735,2746,2770,2777,2788,2795,2805,2815,2832,2854,2868,2898,2904,2911,2927,2948,2962,2472,3221,3224,3225,3228,3229,3231,3233,3235,3237
```

Comment:
@h2nch2co2h is this file needed or is it included by mistake? I noticed this is still using `FPS`.